esmvaltool/_data_finder.py

```diff
@@ -233,7 +233,21 @@ def get_input_fx_dirname_template(variable, rootpath, drs):
         if isinstance(input_dir, six.string_types):
             dir2 = replace_tags(input_dir, variable, i=fx_ind)
         elif _drs in input_dir:
-            dir2 = replace_tags(input_dir[_drs], variable, i=fx_ind)
+            try:
+                insts = cmip5_dataset2inst(variable['dataset'])
+            except KeyError as msg:
+                logger.debug('CMIP5 dataset2inst: %s', msg)
+                insts = 0
+            dirs2 = []
+            if isinstance(insts, list):
+                for j in range(len(insts)):
+                    dir2 = replace_tags(
+                        input_dir[_drs], variable, j=j, i=fx_ind)
+                    dirs2.append(dir2)
```

Comment:
In this line, you collect all `dir2` in the list `dirs2`. However, in the following code you only use `dir2`, not `dirs2`, so the code only returns the fx file for the last `dir2` (i.e. the last institute) found. This could be a problem if not all institutes contain the needed fx file (see #582).

Reply:
very good catch @schlunma ! cheers, I have just fixed it, I also removed the obsolete renaming of seaIce -> ocean since that is removed from https://github.com/ESMValGroup/ESMValTool/pull/581